releasenotes/notes/deprecate-cutqc-7541c514bd96fb05.yaml

```diff
@@ -2,6 +2,6 @@
 deprecations:
   - |
     The ``circuit_cutting.cutqc`` package is deprecated and will be removed no sooner than Circuit Knitting Toolbox 0.8.0. 
-    The wire cutting functionality in the ``circuit_knitting.cutting`` package will be maintained going forward. 
-    Additionally, there is new automated LO gate and wire cut-finding functionality in the ``circuit_knitting.cutting.automated_cut_finding`` module.
-    See the ``docs/tutorials/04_automatic_cut_finding`` notebook for a tutorial on automated LO cut-finding.
+    The wire cutting functionality in the :mod:`circuit_knitting.cutting` package will be maintained going forward. 
+    Additionally, there is new automated LO gate and wire cut-finding functionality in the :mod:`circuit_knitting.cutting.automated_cut_finding` module.
+    A `tutorial <https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/main/docs/circuit_cutting/tutorials/04_automatic_cut_finding.ipynb>`__ has been added to demonstrate automated LO cut-finding.
```

Comment:
Nice

Reply:
Does the tutorial define "LO"?  If not, I'd suggest removing it from this string.

Reply:
Agreed. Done here: [dc31577](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/527/commits/dc315775d8b77de35480026dd9fa0c62f124eeeb)